python/hsfs/feature_group.py

```diff
@@ -46,6 +46,8 @@
 from hsfs.core.job import Job
 from hsfs.core.variable_api import VariableApi
 from hsfs.core import great_expectation_engine
+from hsfs.core.feature_monitoring_config import FeatureMonitoringConfig
+from hsfs.core.feature_monitoring_config_engine import FeatureMonitoringConfigEngine
 
 
 class FeatureGroupBase:
```

Comment:
I guess the `enable_feature_monitoring` method is missing (calling `enable_feature_monitoring_config`in the engine class)